integration-tests/opentelemetry-jdbc-instrumentation/src/main/resources/application.properties

```diff
@@ -7,37 +7,40 @@ quarkus.datasource.devservices.enabled=false
 quarkus.devservices.enabled=false
 
 # JDBC instrumentation setting
-driver=io.opentelemetry.instrumentation.jdbc.OpenTelemetryDriver
+otel-driver=io.opentelemetry.instrumentation.jdbc.OpenTelemetryDriver
```

Comment:
Why do you need that? Wouldn't you just need to enable tracing?

Reply:
I wanted to make sure previous behavior still works. There are 2 datasources using tracing flag and 2 that does not. It verifies backwards compatibility.